### ISSUE TITLE
Dispatching on recalculating observation kernel each inference step/or not

### ISSUE THREAD

**Issue by SamuelBrand1** [CDCgov/Rt-without-renewal#167]
Yes good idea I think this might be a new issue related to #161 (i.e resolve that and then this is a specific instance?)

_Originally posted by @seabbs in https://github.com/CDCgov/Rt-without-renewal/pull/162#discussion_r1533700590_

**Comment by seabbs** [comment]
Can we flesh this out a bit. Its blocked by #161 imo

**Comment by seabbs** [comment]
I think we no longer need this due to #392
